### DISCUSSION TITLE
Consider linking to Binder

### DISCUSSION THREAD

**Discussion by michitux** [networkit/networkit#702] [Ideas]
I have just learned about [Binder](https://mybinder.org/). They provide a platform where you can open Jupyter Notebooks in an executable environment. I have just tried this with NetworKit and - believe me or not - after some minutes I had access to a JupyterHub server where I could execute (and modify) the code in the notebooks included in the documentation. Note that I didn't need to configure anything, this is just based on what is already there in the git repository (primarily `requirements.txt` and `setup.py`). I you don't believe me or want to see it yourself just click on this badge to get to a JupyterHub instance running the current Dev branch of NetworKit: [![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/kit-parco/networkit/Dev) I would suggest to include this badge in the README and on the homepage with an explanation that people can try out NetworKit there.

I think this is awesome for trying out NetworKit without installing it locally, for teaching or also offering reproducible results for papers that don't require too much compute resources (you get at least 1GB RAM and you must be active at least every 10 minutes). From what I have seen the systems seem to be reasonably powerful, e.g. the `TopCloseness.ipynb` notebook had a cell where the included output suggested 22 seconds and it needed 17.5 seconds. Of course, compute resources are not guaranteed.

**Comment by hmeyerhenke** [comment]
Sounds like a really cool thing! Thanks for pointing it out!

**Comment by hmeyerhenke** [comment]
@kolja-esders, can you please take care of this?
